service/history/historyEngine.go

```diff
@@ -145,6 +147,7 @@ func NewEngineWithShardContext(
 	config *Config,
 	replicationTaskFetchers *ReplicationTaskFetchers,
 	domainReplicator replicator.DomainReplicator,
+	clientBean client.Bean,
```

Comment:
remove this line, since we can call `shard.GetService().GetClientBean()`

Reply:
done